### DISCUSSION TITLE
Library Panels: Version History

### DISCUSSION THREAD

**Discussion by henryw374** [grafana/grafana#38767] [Dashboards, Panels & Visualizations]
hi, I would like some info on how to manage version history of library panels. 

Thanks

**Comment by achatterjee-grafana** [comment]
@pkolyvas and @kaydelaney, do we have the capability? If so, can you let me know so that I can update the docs? TY

**Comment by pkolyvas** [comment]
Library panels don't currently have a version history. I'm going to retile this issue and move it to discussions in order to start tracking a feature request.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
